CondFormats/DataRecord/interface/EcalRecHitParametersRcd.h

```diff
@@ -0,0 +1,8 @@
+#ifndef CondFormats_DataRecord_EcalRecHitParametersRcd_h
+#define CondFormats_DataRecord_EcalRecHitParametersRcd_h
+
+#include "FWCore/Framework/interface/EventSetupRecordImplementation.h"
+
+class EcalRecHitParametersRcd : public edm::eventsetup::EventSetupRecordImplementation<EcalRecHitParametersRcd> {};
```

Comment:
Rather than adding a new record, can you reuse `JobConfigurationGPURecord` from `HeterogeneousCore/CUDACore/interface/JobConfigurationGPURecord.h` ?

I will make sure to migrate it from `HeterogeneousCore/CUDACore` to `HeterogeneousCore/AlpakaCore` or some other central place before removing the CUDA code.